internal/distributed/proxy/httpserver/handler_v1.go

```diff
@@ -47,7 +47,7 @@ func (h *Handlers) describeCollection(c *gin.Context, dbName string, collectionN
 		DbName:         dbName,
 		CollectionName: collectionName,
 	}
-	if needAuth {
+	if needAuth && proxy.Params.CommonCfg.AuthorizationEnabled {
```

Comment:
Why needAuth is true if the requested prefix is /vector/collections/describe? yet other requests don't need to check auth.